meta-balena-common/classes/kernel-resin.bbclass

```diff
@@ -0,0 +1 @@
+kernel-balena.bbclass
```

Comment:
I found symlinks didn't work as expected for recipes since they were still including the recipe by the old name but I guess it works for classes?

Reply:
@klutchell works locally for me, if it doesn't work the build check will fail as all device types currently use `kernel-resin` and `inherit` fails if the class is not found.